graphcore_cloud_tools/paperspace_utils/symlink_datasets_and_caches.py

```diff
@@ -313,6 +313,7 @@ def parallel_download_dataset_from_s3(
     symlink=True,
     use_cli=False,
     endpoint_fallback=False,
+    failed_files: List[GradientDatasetFile] = None,
```

Comment:
At this point, these are not failed_files but files that need to be downloaded. Can you rename to reflect?

Reply:
Abandoning this approach